src/components/organisms/Table/v2/Body.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+
+import { getChildrenWithNewProps } from '../../../../helpers'
+
+export const Body: FC = ({ children, ...props }) => {
+  return (
+    <StyledBody>
+      {getChildrenWithNewProps(children, props)}
```

Comment:
Why are you lifting the `Body` props at the children?

Reply:
i didn't understand your questions here. please clarify.

Reply:
`props` contains all the props from `Body`. So why do you give the props from `Body` to its children using `getChildrenWithNewProps(children, props)`? If you give a `className` to `Body`, every direct children would have this class too?

Reply:
but when you do `const Body: FC<TableChildrenProps> = ({ children, ...props })`, it ensures that only the restricted props are getting passed to `Body`. So if `className` is not expected, then TypeScript would complain and won't let it go through.

Reply:
these are replaced with context and reusable components